controllers/imageclusterinstall_controller.go

```diff
@@ -259,8 +259,16 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 		if updateErr := r.setHostConfiguredCondition(ctx, ici, err); updateErr != nil {
 			log.WithError(updateErr).Error("failed to create DataImage")
 		}
-		return ctrl.Result{}, err
```

Comment:
Why remove this? Do we not want to fail here?

Reply:
We do, thanks